django_mongodb_engine/south_adapter.py

```diff
@@ -53,11 +53,11 @@ def alter_column(self, table_name, column_name, field, explicit_name=True):
 
     def delete_column(self, table_name, name):
         collection = self._get_collection(table_name)
-        collection.update({}, {'$unset': {name: 1}})
+        collection.update_many({}, {'$unset': {name: 1}})
```

Comment:
@kondaiahv will this be update_many or update_one?

Reply:
That's a DDL remove column method. So, that's should be update_many to remove a column/field from all docs.